Controller panics when `TRANSFORMER_IMAGE` environment variable is not set

The following panic occurs when the `TRANSFORMER_IMAGE` environment variable is not set:

```
panic: unable to process Transformation required environment variables (missing TRANSFORMER_IMAGE)

goroutine 1 [running]:
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc000234240, {0x0, 0x0, 0x0})
        go.uber.org/zap@v1.19.0/zapcore/entry.go:232 +0x446
go.uber.org/zap.(*SugaredLogger).log(0xc000362760, 0x4, {0x0, 0x2}, {0xc000ab5848, 0x3, 0x1a}, {0x0, 0x0, 0x0})
        go.uber.org/zap@v1.19.0/sugar.go:227 +0xee
go.uber.org/zap.(*SugaredLogger).Panic(...)
        go.uber.org/zap@v1.19.0/sugar.go:123
github.com/triggermesh/triggermesh/pkg/transformation/reconciler/controller.NewController({0x2ab7e08, 0xc000a5eea0}, {0xc000a55ea0, 0xc00070c840})
        github.com/triggermesh/triggermesh/pkg/transformation/reconciler/controller/controller.go:61 +0x26d
knative.dev/pkg/injection/sharedmain.ControllersAndWebhooksFromCtors({0x2ab7e08, 0xc000a5eea0}, 0x1, {0xc000ab5d90, 0x3c, 0x77359400})
        knative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:350 +0x15c
knative.dev/pkg/injection/sharedmain.MainWithConfig({0x2ab8ce8, 0xc00000e030}, {0x26ccba1, 0x16}, 0xc00057c6c0, {0xc000aa1d90, 0x3c, 0x3c})
        knative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:192 +0x5da
knative.dev/pkg/injection/sharedmain.MainWithContext({0x2ab8ce8, 0xc00000e030}, {0x26ccba1, 0x16}, {0xc00089fd90, 0x3c, 0x3c})
        knative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:131 +0xe7
knative.dev/pkg/injection/sharedmain.Main({0x26ccba1, 0x16}, {0xc00089fd90, 0x3c, 0x3c})
        knative.dev/pkg@v0.0.0-20210919202233-5ae482141474/injection/sharedmain/main.go:105 +0x8f
main.main()
        github.com/triggermesh/triggermesh/cmd/triggermesh-controller/main.go:80 +0x3d4
```

We should instead default to something, like the public "latest" image name, because most `ko://` entries may be commented during development.

The culprit:

https://github.com/triggermesh/triggermesh/blob/b4b1d5dda573a2ffcc335272a9e05a64272589b1/pkg/transformation/reconciler/controller/controller.go#L56-L62

--- a/pkg/transformation/reconciler/controller/controller.go
+++ b/pkg/transformation/reconciler/controller/controller.go
@@ -35,7 +35,7 @@ import (
 )
 
 type envConfig struct {
-	Image string `envconfig:"TRANSFORMER_IMAGE" required:"true"`
+	Image string `envconfig:"TRANSFORMER_IMAGE" default:"gcr.io/triggermesh/transformation-adapter"`
 }
 
 // NewController creates a Reconciler and returns the result of NewImpl.
@@ -54,12 +54,7 @@ func NewController(
 	}
 
 	env := &envConfig{}
-	if err := envconfig.Process("", env); err != nil {
-		logger.Panicf("unable to process Transformation required environment variables: %v", err)
-	}
-	if env.Image == "" {
-		logger.Panic("unable to process Transformation required environment variables (missing TRANSFORMER_IMAGE)")
-	}
+	envconfig.MustProcess("", env)
 
 	r.transformerImage = env.Image
 

